src/github-client.js

```diff
@@ -40,6 +40,31 @@ function githubClient(github, context) {
       return data
     },
 
+    async enableAutoMergePullRequest(pullRequestId, mergeMethod) {
+      const query = `
+mutation ($pullRequestId: ID!, $mergeMethod: PullRequestMergeMethod!) {
+  enablePullRequestAutoMerge(
+    input: { pullRequestId: $pullRequestId, mergeMethod: $mergeMethod }
+  ) {
+    pullRequest {
+      autoMergeRequest {
+        enabledAt
+        enabledBy {
+          login
+        }
+      }
+    }
+  }
+}
+`
+      const { data } = await github.graphql(query, {
+        pullRequestId,
+        mergeMethod: mergeMethod.toUpperCase(),
+      })
+      // todo assert
```

Comment:
@shouichi are you planning to do something here?

Reply:
Removed it.